packages/advisor-components/src/ReportDetails/ReportDetails.spec.ct.js

```diff
@@ -65,6 +65,7 @@ describe('report details: kba loaded', () => {
   });
 
   it('renders a loaded kba link', () => {
+    cy.mount(<ReportDetails {...{ ...props, kbaLoading: true, isProd: true }} />);
```

Comment:
You are calling cy.mount twice (one in beforeEach, and one in `it`). It's not healthy for the test, and we better keep only one mount command. Can we just add this prop in cy.mount in beforeEach above?